reg_tests/executePythonRegressionCase.py

```diff
@@ -25,11 +25,12 @@
 
 import os
 import sys
-basepath = os.path.sep.join(sys.argv[0].split(os.path.sep)[:-1]) if os.path.sep in sys.argv[0] else "."
+basepath = os.path.dirname(__file__)
```

Comment:
Could we add the `abspath` here, like the other scripts (see https://github.com/OpenFAST/openfast/pull/1021)?